polaris-core/src/main/java/org/apache/polaris/core/PolarisConfiguration.java

```diff
@@ -103,6 +103,22 @@ public static <T> Builder<T> builder() {
               .defaultValue(false)
               .build();
 
+  // Config key for whether to skip credential-subscoping indirection entirely whenever trying
+  // to obtain storage credentials for instantiating a FileIO. If 'true', no attempt is made
+  // to use StorageConfigs to generate table-specific storage credentials, but instead the default
+  // fallthrough of table-level credential properties or else provider-specific APPLICATION_DEFAULT
+  // credential-loading will be used for the FileIO.
+  // Typically this setting is used in single-tenant server deployments that don't rely on
+  // "credential-vending" and can use server-default environment variables or credential config
+  // files for all storage access, or in test/dev scenarios.
+  public static final Boolean SKIP_CREDENTIAL_SUBSCOPING_INDIRECTION_DEFAULT = false;
```

Comment:
I would rather not have a separate public variable here; you can always get the default with `SKIP_CREDENTIAL_SUBSCOPING_INDIRECTION.defaultValue`